src/Components/Shared/src/ArrayBuilder.cs

```diff
@@ -6,7 +6,11 @@
 using System.Diagnostics;
 using System.Runtime.CompilerServices;
 
+#if COMPONENTS_SERVER
+namespace Microsoft.AspNetCore.Components.Web.Rendering
+#else
 namespace Microsoft.AspNetCore.Components.RenderTree
+#endif
```

Comment:
Do the namespaces clash because of `InternalsVisibleTo`?

Reply:
The IgnitorClient has IVT to both these assemblies. We need to fix how it references Components, but this seemed like a less problematic solution.